Picard_inputs/KMT2A_exon_processing.R

```diff
@@ -0,0 +1,13 @@
+setwd("~/new_projects/tandemhunter_coverage")
```

Comment:
You might need to change the set working directory line ('setwd') to a directory that is available when you run TandemHunter as an app, as I can't see 'new_projects' mentioned or created anywhere.
R nitpick: Some <- arrows and = signs don't have spaces before them when they should.